WordPress/src/main/java/org/wordpress/android/ui/reader/viewmodels/ReaderPostDetailViewModel.kt

```diff
@@ -179,6 +193,22 @@ class ReaderPostDetailViewModel @Inject constructor(
         }
     }
 
+    fun onRelatedPostsRequested(sourcePost: ReaderPost) {
+        /* Related posts only available for wp.com */
+        if (!sourcePost.isWP) return
+
+        launch {
+            when (val fetchRelatedPostsState = readerFetchRelatedPostsUseCase.fetchRelatedPosts(sourcePost)) {
+                is FetchRelatedPostsState.AlreadyRunning,
+                is FetchRelatedPostsState.Failed.NoNetwork,
+                is FetchRelatedPostsState.Failed.RequestFailed -> { // Do Nothing
```

Comment:
Minor (🔍): Instead of using braces, I usually do this trick to get to a one-liner result. I am sharing here in case you might like it as well:
```
is FetchRelatedPostsState.Failed.RequestFailed -> Unit // Do Nothing
```

PS: I actually created a helper function on my previous roles, like a `doNothing`, which is an alias to `Unit` and this way it can end up even easier to read and apprehend, without the need for commenting:

```
is FetchRelatedPostsState.Failed.RequestFailed -> doNothing
```

Reply:
Nice suggestion, in c4bcf98.